src/components/FilterTags/FilterTags.js

```diff
@@ -87,30 +88,89 @@ const FilterTags = () => {
     };
   }, []);
 
-  const handleFilterClick = () => {
-    console.log("Filter clicked");
+  /**
+  Apply a given filter
+  @param {object} e The event object
+  @param {string} filter The name of the filter
+   @param {string} filterBy The name of the filter group
+*/
+  const addActiveFilter = filter => {
+    setActiveFilters(filters => {
+      const updatedFilters = [...filters];
+      if (!updatedFilters.includes(filter)) {
+        updatedFilters.push(filter);
+      }
+      return updatedFilters;
+    });
+  };
+
+  /**
+  Remove a given filter
+  @param {object} e The event object
+  @param {string} filter The name of the filter
+   @param {string} filterBy The name of the filter group
+*/
+  const removeActiveFilter = filter => {
+    setActiveFilters(filters => {
+      const updatedFilters = [...filters];
+      if (updatedFilters.includes(filter)) {
+        const index = updatedFilters.indexOf(filter);
+        updatedFilters.splice(index, 1);
+      }
+      return updatedFilters;
+    });
   };
 
   return (
     <div className="p-filter-tags" ref={node}>
-      <form>
-        <input
-          type="text"
-          placeholder="Filter models"
-          className="p-filter-tags__input"
-          onFocus={() => setfilterPanelVisibility(true)}
-        />
-      </form>
+      <input
+        type="text"
+        className="p-filter-tags__input"
+        onClick={() => setFilterPanelVisibility(!filterPanelVisibility)}
+        placeholder={
+          activeFilters.length
+            ? `Active filters: ${activeFilters.length}`
+            : "Filter models"
+        }
+      />
+
       <div
         className={classNames("p-card--highlighted p-filter-panel", {
           "is-visible": filterPanelVisibility
         })}
       >
         {Object.entries(filters).length <= 0 && <p>Loading filters...</p>}
+
+        {activeFilters.length > 0 && (
```

Comment:
I feel like this section should always be visible as it jumps quite a bit the first time it becomes active. It also acts as a hint to the user what to do. It could even have a message when empty "Click a filter" or something. @clagom

Reply:
Ok, I'll add it in all the time and @clagom can review on staging..